backend/server.js

```diff
@@ -28,4 +29,6 @@ app.use('/graphql', graphqlHttp(req => ({
   context: req
 })))
 
+router.post('/upload', imageUploader)
```

Comment:
We might want endpoints to fetch images and/or download images, so I'd namespace this route to `/static/upload`.